tasks/windows.json

```diff
@@ -0,0 +1,18 @@
+{
+  "description": "Manage the state of Windows services (without a puppet agent)",
+  "input_method": "powershell",
+  "parameters": {
+    "action": {
```

Comment:
I'm torn on the name `action`. My first expectation from prior experience is that this should be `state`, as everything I've ever used goes that way. This does not account for 'status', but I am used to separate commands for getting status and setting status. I am unaware of what the current conventions for using tasks, so ignore if the other tasks use this convention

Reply:
Yeah, not sure how much thought was given to `action`, but that was already in use for the existing task, so I carried it forward - https://github.com/puppetlabs/puppetlabs-service/blob/master/tasks/init.json#L5

Just checked on package tasks and they follow the same structure - https://github.com/puppetlabs/puppetlabs-package/blob/master/tasks/init.json#L5-L8